core/jazz_codeq/index.js

```diff
@@ -173,14 +173,31 @@ function getReportOnError(err, metrics, config, serviceContext) {
 	let result = {};
 	if (err && err.report_error) {
 		if (err.code === 404) {
-			return utils.getReport(metrics, null, config)
+			logger.info('404 error'+err)
+		const result_inputs = getCodeqInputsUsingQuery(serviceContext, config);
+		logger.info('result_inputs',result_inputs)
+
+		if (result_inputs.error) {
+			const errorMessage = (typeof result_inputs.error === 'object') ? JSON.stringify(errorHandler.throwInputValidationError(result_inputs.error)) : result_inputs.error;
+			logger.error(errorMessage);
+			return cb(errorMessage);
+		} else {
+			const metrics = result_inputs.metrics;
+			const query = result_inputs.query;
+		}
+
+		return utils.getJazzToken(config)
```

Comment:
why?

Reply:
these changes were not required, have reverted it